policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/service/PolicyServiceCachingProxy.java

```diff
@@ -56,7 +56,7 @@ public PolicyServiceCachingProxy(final PolicyService service) {
      */
     @Cacheable(value = "resourceRules", key = "#resourceId")
     public Optional<Rules<LeafResource>> getResourceRules(final String resourceId) {
-        LOGGER.info("Cache miss for resourceId {}", resourceId);
+        LOGGER.debug("Cache miss for resourceId {}", resourceId);
```

Comment:
Would it be better to make this log message different to the `getRecordRules` message? This would help to know what was being done when going through the logs

Reply:
Added cache name to each